oxide/lib_test.go

```diff
@@ -144,3 +155,58 @@ func Test_buildRequest(t *testing.T) {
 		})
 	}
 }
+
+func Test_Client(t *testing.T) {
+	tt := map[string]struct {
+		config    *Config
+		env       map[string]string
+		wantError bool
```

Comment:
If we make `wantError` a string we can verify it's contents are what we expect them to be
```suggestion
		wantError string
```